components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -357,8 +356,10 @@ void MAMMicrophysics::run_impl(const double dt) {
       ekat::subview(w_updraft, icol),
       pblh_(icol));
 
+    // set surface state data
+    haero::Surface sfc{};
```

Comment:
I don't believe MAM4's nucleation process interacts with the surface directly at all, so we just create a "blank" `Surface` representation here to pass in to the process, as required by its interface. Down the line, this surface representation will actually need to be filled with data that is used by other aerosol processes. Since all these processes have the same interface, the Surface gets passed to each one whether or not it actually is used. The uniformity of the interface makes it easier to change the implementation/underlying model in an aerosol process without cascading code changes.